main.py

```diff
@@ -16,12 +17,21 @@ def sort_list(items, ascending=True):
     return sorted(items, reverse=(not ascending))
 
 
+def remove_duplicates_from_list(items):
+    return list(set(items))
```

Comment:
¿Podrías añadir una comprobación de tipo?

```suggestion
def remove_duplicates_from_list(items):
    if not isinstance(items, list):
        raise RuntimeError(f"No pueden quitar duplicados de {type(items)}")
    return list(set(items))
```